data/mods/Dark-Skies-Above/items/electronics.json

```diff
@@ -1,58 +1,216 @@
+    "name": { "str": "broken xenoid emissary of war", "str_pl": "broken emissaries of war" },
     "category": "other",
-    "description": "The massive body of a collapsed emissary of flame.  Still a bit intimidating, perhaps knowing the damage it can cause.  Could be gutted for parts, but you'll probably need specialized alien tools.",
+    "description": "The massive body of a collapsed emissary of war.  Still extremely intimidating, perhaps knowing the damage it can cause.  Could be gutted for parts, but you'll probably need specialized alien tools.",
     "price": 1000,
-    "material": [ "superalloy" ],
+    "material": [ "dkz_biogel_mat", "dkz_blend_mat", "dkz_biopoly_mat" ],
     "volume": "875000 ml",
     "weight": "200 kg",
-    "flags": [ "TRADER_AVOID", "NO_REPAIR" ]
+    "flags": [ "TRADER_AVOID", "NO_REPAIR", "LEAK_DAM", "RADIOACTIVE" ]
   },
   {
     "type": "GENERIC",
-    "id": "broken_dks_glowdrone",
+    "id": "broken_dkz_emissary_flame",
+    "symbol": ",",
+    "color": "green",
+    "name": { "str": "broken xenoid emissary of flame", "str_pl": "broken emissaries of flame" },
+    "category": "other",
+    "description": "The massive body of a collapsed emissary of flame.  Still very intimidating, perhaps knowing the damage it can cause.  Could be gutted for parts, but you'll probably need specialized alien tools.",
+    "price": 1000,
+    "material": [ "dkz_biogel_mat", "dkz_blend_mat", "dkz_biopoly_mat" ],
+    "volume": "875000 ml",
+    "weight": "200 kg",
+    "flags": [ "TRADER_AVOID", "NO_REPAIR", "LEAK_DAM", "RADIOACTIVE" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "broken_dkz_glowdrone",
     "copy-from": "broken_eyebot",
-    "name": { "str": "broken surveillance drone" },
+    "name": { "str": "broken xenoid reconnaissance drone" },
+    "material": [ "dkz_biogel_mat", "dkz_blend_mat", "dkz_biopoly_mat" ],
+    "flags": [ "TRADER_AVOID", "NO_REPAIR", "LEAK_DAM", "RADIOACTIVE" ],
     "description": "A broken drone.  Much less threatening now that it isn't shining its light everywhere.  Could be gutted for parts.  Specialized alien tools would be best for disassembly, but you could make do with more human instruments instead."
   },
   {
     "type": "GENERIC",
-    "id": "broken_dks_scidrone",
+    "id": "broken_dkz_scidrone",
     "copy-from": "broken_eyebot",
-    "name": { "str": "broken seeker drone" },
+    "name": { "str": "broken xenoid seeker drone" },
+    "material": [ "dkz_biogel_mat", "dkz_blend_mat", "dkz_biopoly_mat" ],
+    "flags": [ "TRADER_AVOID", "NO_REPAIR", "LEAK_DAM", "RADIOACTIVE" ],
     "description": "A broken drone.  Much less threatening now that it isn't prodding you.  Specialized alien tools would be best for disassembly, but you could make do with more human instruments instead."
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "id": "dkz_zenture_industries_basic_zwatch",
+    "name": { "str": "Zenture Industries® z-Ŵatch™", "str_pl": "Zenture Industries® z-Ŵatches™" },
```

Comment:
What is Zenture Industries? Why do they have 3 watches? Why is it so advance and "cool" that it the description goes out of its way to state how cool it is, how user friendly it is, and how advanced it is? And who is Mr. Zenture and team zenture?